renderers/games/teams.py

```diff
@@ -78,11 +93,29 @@ def __render_team_text(canvas, layout, colors, team, homeaway, full_team_names,
     graphics.DrawText(canvas, font["font"], coords["x"], coords["y"], text_color_graphic, team_text)
 
 
-def __render_team_score(canvas, layout, colors, runs, homeaway, default_colors):
+def __render_score_component(canvas, layout, colors, homeaway, default_colors, coords, component_val, width):
+    # The coords passed in are the rightmost pixel.
+    RHE_SPACING = 3  # Number of pixels between runs/hits and hits/errors.
```

Comment:
Can we make this either dependent on the font size or part of the config?

Reply:
Thanks - I appreciate how you push me toward simplicity.  I got rid of the extra logic around 1's and made spacing dependent on font size.  However I do like the reduced spacing between digits - it makes a visible difference especially because we're not using a full character's space between scores. 

What I've come up with is hopefully a compromise - a "compact" option - it looks good with 1 and ok with 2.  This way it's also possible to compare the two approaches.  Note that the configuration has changed. Let me know what you think!

Unfortunately black is now spreading some of the function calls across many lines.  Too many parameters, I guess.

Reply:
A nice compromise. I think this compact setting should live in the coordinate .jsons, not the global config. That way it can place nice with different sizes. We should update all the example sizes with a reasonable default - probably false on all the larger sizes, but it's worth seeing how it looks first

Reply:
That's certainly a possibility.  I didn't go this route because I think the coordinates file just contain x,y coodinates, and this is more of a spacing option.  Also my thought was that most people who implement this will be doing so for only one size, but I guess you never know. 

So in that case should we put BOTH settings in the layout? Perhaps something like this... 

runs_hits_errors {
show: true
compact: true
}

I like the idea of just using the "runs" coordinate as the location, but we could instead put "home" and "away" coordinates to the above.  What do you think?

Reply:
We already have flags in the coordinates in a few places (one of which was even there before my tinkering):

https://github.com/WardBrian/mlb-led-scoreboard/blob/9c8cbf0f1bb6f86e509ab47cf004b22345572f0a/coordinates/w64h32.json.example#L56-L57
https://github.com/WardBrian/mlb-led-scoreboard/blob/9c8cbf0f1bb6f86e509ab47cf004b22345572f0a/coordinates/w64h32.json.example#L198

So I think this is fine. Let's just put it in Runs, no reason to split home/away. I'd change 'compact' to something like 'compress_digits' or similar, but other than that it sounds good to me.